test_version/version_specific_tests_dir/twitter_scrooge/twitter_scrooge_bindings.bzl

```diff
@@ -0,0 +1,37 @@
+load("@io_bazel_rules_scala//twitter_scrooge:twitter_scrooge.bzl", "defaulted_twitter_scrooge_dependency", "twitter_scrooge")
```

Comment:
this is the example of how to override and customize scrooge, right?
should we add a comment for this? Might be obvious right now but maybe beneficial for future pop in visitors

Reply:
Good point! Done :)